code/modules/crafting/recipes.dm

```diff
@@ -606,3 +606,13 @@
 	name = "Mummification Bandages (Body)"
 	result = /obj/item/clothing/under/mummy
 	reqs = list(/obj/item/stack/sheet/cloth = 5)
+
+/datum/crafting_recipe/guillotine
+	name = "Guillotine"
+	result = /obj/structure/guillotine
+	time = 150 // Building a functioning guillotine takes time
+	reqs = list(/obj/item/stack/sheet/plasteel = 3,
```

Comment:
Plasteel is a bit much I think.  Perhaps change it to normal metal?

Reply:
nitpicking, it's fineee